mri/reconstructors/utils/extract_sensitivity_maps.py

```diff
@@ -158,14 +158,15 @@ def get_Smaps(k_space, img_shape, samples, thresh,
                                   endpoint=False)
                       for i in np.arange(np.size(img_shape)-1)]
         grid = np.meshgrid(*grid_space)
-        kspace_plane_loc, _, sort_pos, idx_mask_z = \
+        kspace_plane_loc, z_sample_loc, sort_pos, idx_mask_z = \
```

Comment:
Is `z_sample_loc` needed here?

Reply:
Ahh, with new function definition, no.. Earlier we did need it.

Reply:
idx_mask_z holds these locations